deployments/examples/ocis_individual_services/docker-compose.yml

```diff
@@ -540,49 +701,45 @@ services:
       ocis-net:
     entrypoint:
       - ocis
-      - storage-public-link
+      - storage-publiclink
       - server
     environment:
-      OCIS_LOG_LEVEL: "${OCIS_LOG_LEVEL:-error}"
-      OCIS_LOG_COLOR: "${OCIS_LOG_COLOR:-false}"
-      OCIS_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
+      STORAGE_PUBLICLINK_LOG_LEVEL: "${OCIS_LOG_LEVEL:-error}"
+      STORAGE_PUBLICLINK_LOG_COLOR: "${OCIS_LOG_COLOR:-false}"
+      STORAGE_PUBLICLINK_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
 
-      STORAGE_PUBLIC_LINK_ENDPOINT: storage-publiclink:9178
-      STORAGE_PUBLIC_LINK_GRPC_ADDR: 0.0.0.0:9178
+      STORAGE_PUBLICLINK_GRPC_ADDR: 0.0.0.0:9178
 
-      STORAGE_JWT_SECRET: ${OCIS_JWT_SECRET:-Pive-Fumkiu4}
-      REVA_GATEWAY: storage-gateway:9142
+      STORAGE_PUBLICLINK_JWT_SECRET: ${OCIS_JWT_SECRET:-Pive-Fumkiu4}
+      REVA_GATEWAY: gateway:9142
     logging:
       driver: "local"
     restart: always
 
-  storage-sharing:
+  sharing:
     image: owncloud/ocis:${OCIS_DOCKER_TAG:-latest}
     deploy:
       replicas: 1
     networks:
       ocis-net:
     entrypoint:
       - ocis
-      - storage-sharing
+      - sharing
       - server
     environment:
-      OCIS_LOG_LEVEL: "${OCIS_LOG_LEVEL:-error}"
-      OCIS_LOG_COLOR: "${OCIS_LOG_COLOR:-false}"
-      OCIS_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
+      SHARING_LOG_LEVEL: "${OCIS_LOG_LEVEL:-error}"
+      SHARING_LOG_COLOR: "${OCIS_LOG_COLOR:-false}"
+      SHARING_LOG_PRETTY: "${OCIS_LOG_PRETTY:-false}"
 
-      STORAGE_SHARING_ENDPOINT: storage-sharing:9150
-      STORAGE_SHARING_GRPC_ADDR: 0.0.0.0:9150
+      SHARING_GRPC_ADDR: 0.0.0.0:9150
 
-      STORAGE_SHARING_USER_DRIVER: json
-      STORAGE_SHARING_USER_JSON_FILE: /var/lib/ocis/storage/sharing/shares.json
-      STORAGE_SHARING_PUBLIC_DRIVER: json
-      STORAGE_SHARING_PUBLIC_JSON_FILE: /var/lib/ocis/storage/sharing/publicshares.json
+      SHARING_EVENTS_ENDPOINT: nats:9233
 
-      STORAGE_SHARING_EVENTS_ADDRESS: nats:9233
+      SHARING_JWT_SECRET: ${OCIS_JWT_SECRET:-Pive-Fumkiu4}
+      REVA_GATEWAY: gateway:9142
 
-      STORAGE_JWT_SECRET: ${OCIS_JWT_SECRET:-Pive-Fumkiu4}
-      REVA_GATEWAY: storage-gateway:9142
+      SHARING_USER_DRIVER: json
+      SHARING_PUBLIC_DRIVER: json
     volumes:
       - "ocis-storage-sharing:/var/lib/ocis"
```

Comment:
the volume should follow the service name for consistence reasons (without "storage")